seeder/src/seeder_flet/lib/views/login.py

```diff
@@ -61,7 +64,7 @@ async def login_view():
                             color={ft.MaterialState.DEFAULT: ft.colors.BLUE_100},
                             bgcolor=ft.colors.TERTIARY,
                         ),
-                        on_click=lambda e: handle_login_click(
+                        on_click=lambda e: handle_login_click_wrapper(
```

Comment:
https://stackoverflow.com/questions/40746213/how-to-use-await-in-a-python-lambda